JFS_CAN_report_instructions.qmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "JFS CAN Data Report Instructions"
+format: pdf
```

Comment:
What do you think about making the directions html?  I think we could host it on the "pages" site here so that Kris (or whoever will run it) will always have the latest version at hand.  Also, if the user is in the browser, it might make it easier to flip between instructions and the shiny starter app running in the browser.

Reply:
Yeah, that's a good idea. I'll switch it over